mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx

```diff
@@ -98,15 +112,42 @@ export function DexNavigator (): JSX.Element {
         component={DexScreen}
         name='DexScreen'
         options={{
+          ...screenOptions,
+          headerLeft: undefined,
+          headerLeftContainerStyle: null,
+          headerTitleAlign: 'left',
+          headerBackTitleVisible: false,
+          headerTitleContainerStyle: tailwind('mt-4 ml-5'),
+          headerRightContainerStyle: [screenOptions.headerRightContainerStyle, tailwind('mt-5 justify-start', { 'pr-3': Platform.OS !== 'ios' })],
```

Comment:
does this works? works on my both devices
```suggestion
          headerRightContainerStyle: [screenOptions.headerRightContainerStyle, tailwind('mt-5 justify-start', { 'pr-5': Platform.OS !== 'ios' })],
```

Reply:
are we not including this change? without it the header network is mis-aligned on my android @fullstackninja864

Reply:
Updated in recent commit